<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VChar64</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="BigChar" name="bigchar" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>270</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Char Index</string>
    </property>
    <property name="buddy">
     <cstring>spinBox</cstring>
    </property>
   </widget>
   <widget class="CharSetView" name="charsetview" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>731</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Colors</string>
    </property>
    <widget class="ColorPalette" name="colorPalette" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>206</y>
       <width>192</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MultiColor Mode</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multi Color #1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multi Color #2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Foreground</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ColorRect" name="colorRect_0" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="ColorRect" name="colorRect_1" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="ColorRect" name="colorRect_2" native="true"/>
      </item>
      <item row="3" column="2">
       <widget class="ColorRect" name="colorRect_3" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionInvert"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open" resource="../res/resources.qrc">
     <normaloff>:/icons/22x22/actions/document-open.png</normaloff>:/icons/22x22/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="toolTip">
    <string>Open VChar64 project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icons/22x22/actions/document-save.png</normaloff>:/icons/22x22/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy" resource="../res/resources.qrc">
     <normaloff>:/icons/22x22/actions/edit-copy.png</normaloff>:/icons/22x22/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>:/images/24x24/edit-paste.png</normaloff>:/images/24x24/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
   <property name="toolTip">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save" resource="../res/resources.qrc">
     <normaloff>:/icons/22x22/actions/document-save.png</normaloff>:/icons/22x22/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
   <property name="toolTip">
    <string>Save Project As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset theme="document-new" resource="../res/resources.qrc">
     <normaloff>:/icons/22x22/actions/document-new.png</normaloff>:/icons/22x22/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
   <property name="toolTip">
    <string>Invert character</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BigChar</class>
   <extends>QWidget</extends>
   <header>bigchar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharSetView</class>
   <extends>QWidget</extends>
   <header>charsetview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPalette</class>
   <extends>QWidget</extends>
   <header>colorpalette.h</header>
   <container>1</container>
   <slots>
    <signal>colorSelected()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ColorRect</class>
   <extends>QWidget</extends>
   <header>colorrect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bigchar</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>charsetview</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>colorPalette</sender>
   <signal>colorSelected()</signal>
   <receiver>bigchar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPalette</sender>
   <signal>colorSelected()</signal>
   <receiver>colorRect_0</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPalette</sender>
   <signal>colorSelected()</signal>
   <receiver>colorRect_1</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPalette</sender>
   <signal>colorSelected()</signal>
   <receiver>colorRect_2</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPalette</sender>
   <signal>colorSelected()</signal>
   <receiver>colorRect_3</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>colorPalette</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>colorPalette</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>colorPalette</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4</sender>
   <signal>clicked()</signal>
   <receiver>colorPalette</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
